test.sh

```diff
@@ -40,8 +40,17 @@ echo "________Validate chainspecs________"
 fi
 
 
-# Running test's
+# Running tests
 echo "________Running Parity Full Test Suite________"
-
 cargo test -j 8 $OPTIONS --features "$FEATURES" --all $1
 
+# Running the C example
```

Comment:
maybe move to dedicated shell script to keep `test.sh` clean?

Reply:
I think that having two shell scripts makes it work than just one (especially since the script is only 8 lines). But if you have a strong opinion on this I will do it.